src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let totalAmount = 0;
+
+  if (days > 1 && days < 3) {
+    totalAmount = days * 40;
```

Comment:
what is days * 40?
You repeat it 3 times
Move this to the variables
Also, try to avoid else if